src/app/app.component.html

```diff
@@ -7,7 +7,6 @@
     yAxisLabel="Total cases by country"
     xAxisLabel="Days since {{ graphDefinitions[i].cutoff }} cases"
     [data]="graph"
-    [animate]="true"
```

Comment:
I don't actually find the animations useful 😅

We can add them as a URL param if you'd like!

Reply:
Re-added as URL param: https://github.com/prendradjaja/covid-dashboard/pull/13/commits/248ea88735be0daf962c962aaee0b3d43dc21ded